### ISSUE TITLE
Fix all ESlint warnings

### ISSUE THREAD

**Issue by nt1m** [nt1m/devtools-prototyper#45]
There are 2 main warnings : 
- no-unused vars : ESlint doesn't seem to support React properly, we can either ignore those warnings, or disable the ESlint rule
- strict : ESlint wants *all* the files to have "use strict";, what do you think @mdibaiee ?

**Comment by mdibaiee** [comment]
about no-unused-vars I think we might want to disable that, if we were using modules, it would be ok as eslint understands exports.

I only see one warning about `strict` which says `main.jsx` should not have a `"use strict"` directive. Other than that, I don't see any errors. Are you running `grunt eslint`?

**Comment by nt1m** [comment]
@mdibaiee Yep, I'm running `grunt eslint`. And it displays the warning for all the files not having the "use strict" header.

**Comment by nt1m** [comment]
About 25 warnings are dedicated to `strict`

**Comment by mdibaiee** [comment]
@nt1im that was fast :grin:

This is the result of `grunt eslint` for me: https://gist.github.com/mdibaiee/9d0a918ab57653d01262

**Comment by nt1m** [comment]
@mdibaiee I should be sleeping right now ;)
Here's my `grunt eslint` result : https://gist.github.com/nt1m/5d7cca0d4bbf73474b8c (I have some changes locally, but they only give 1 or 2 extra warnings)
Maybe we have different versions of eslint ?

**Comment by mdibaiee** [comment]
I'm using version `0.24.1`. I couldn't sleep.

**Comment by nt1m** [comment]
@mdibaiee Me too, I wonder what's going on.
